students/conferences.md

```diff
@@ -9,148 +9,204 @@ current: students
+We also have in mind that it will probably be the first public talk for you. Scary thought, right? We know. Stage fright is something we all experience, even if we already have held a lot of talks. We will help you in dealing with it! Here’s a start: <a href="http://railsgirlssummerofcode.org/blog/2014-07-29-talk-tips">How to ace your lightning talk</a>.
 
 <h2 id="travel">What About Travel and Accommodation?</h2>
-Unfortunately, we are not able to provide additional financial support for travel & accommodation. If we see an opportunity, we will ask the conference organizers about it. But please consider that providing travel grants is a big effort for everyone involved and not usually happening.
+Unfortunately, we are not able to provide additional financial support for travel & accommodation. If we see an opportunity, we will ask conference organizers about it, but please consider that providing travel grants is a big effort for everyone involved and happens not that often.
 
 Many companies support their employees in attending conferences. Ask your coaches if there’s an opportunity at your place!
 
 <h2 id="organizers">Infos for Conference Organizers</h2>
 You are organizing a conference and want to provide free tickets for Rails Girls Summer of Code students? We will forever keep you in our hearts!
 
-Our students are located all over the world. 2015 teams mainly work on projects using Ruby/Rails/Ruby-related frameworks. We now also have a few JS projects. But we are also looking for general topic developer conferences or full-stack conferences.
+Our students are located all over the world. 2016 teams work on various projects using Ruby/Python/JS and other languages. But we are also looking for general topic developer conferences or full-stack conferences.
 
 There are a few things to consider though:
 
 * Your conference should be beginner friendly. That means, talks (or workshops) need to be suitable for people who haven’t done this for years.
 
-* You take care of a welcoming, respectful and harassment-free atmosphere. We expect you to either have a Code of Conduct in place or otherwise make an effort in dealing with - let us be plain - jerk behavior. Bonus points go to conferences who have assigned contact persons for potential incidents.
-
-At the conference, we want our students to hold a lightning talk about their project. If you have slots for this, we’d be very happy if a team could use one of them. If not, no worries. Alternatively, they can write about their conference experience on our blog.
+* You take care of a welcoming, respectful and harassment-free atmosphere. We expect you to either have a Code of Conduct in place or otherwise make an effort in dealing with -- let us be plain -- jerk behavior. Bonus points go to conferences who have assigned contact persons for potential incidents.
```

Comment:
We now don't accept conferences without a code of conduct, so it should be made clearer here. I suggest replacing this sentence with:

"We expect you to have a Code of Conduct in place and to make an effort in dealing with -- let us be plain -- jerk behavior."